dsl/jpql/src/main/kotlin/com/linecorp/kotlinjdsl/dsl/jpql/Jpql.kt

```diff
@@ -366,6 +366,22 @@ open class Jpql : JpqlDsl {
         )
     }
 
+    /**
+     * Creates an expression that represents the rounding of the specified property's value to a specified scale.
+     */
+    @SinceJdsl("3.4.0")
+    fun <T : Any, V : Number> round(expr: KProperty1<T, @Exact V>, scale: Expression<Int>): Expression<Double> {
+        return Expressions.round(Paths.path(expr), scale.toExpression())
+    }
+
+    /**
+     * Creates an expression that represents the rounding of the specified property's value to a specified scale.
+     */
+    @SinceJdsl("3.4.0")
+    fun <T : Number> round(value: Expressionable<T>, scale: Expressionable<Int>): Expression<Double> {
```

Comment:
In the [jakarta specification](https://jakarta.ee/specifications/persistence/3.1/jakarta-persistence-spec-3.1.html#a5284), the return type of `ROUND` is the same type as the first argument: `value`. 

Could you test this to make sure it's really correct? I think it would be okay to test with Int, Double, and BigDecimal only.

Reply:
![image](https://github.com/line/kotlin-jdsl/assets/31586979/75ff4afc-4e5d-405d-888d-d00a7a3c5f77)